css-two-column-layout/styles.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  font-family: sans-serif;
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-content: center;
```

Comment:
I would move these properties to their own class and apply them as need. a typical `row` class just needs display flex on it, as it is only responsible for holding a collection of column class elements. it is better as projects scale to differentiate between a `layout` class and a class you are using to style a particular element.